appmap/test/test_django.py

```diff
@@ -48,3 +48,85 @@ def test_http_capture(events):
         'status_code': 404,
         'mime_type': 'text/html'
     }
+
+
+def test_message_capture_get(events):
+    client = django.test.Client()
+    client.get('/test', { 'my_param': 'example' }, content_type='application/x-www-form-urlencoded')
```

Comment:
`GET` requests shouldn't take `content_type` as it relates to body, which GET requests lack.